scripts/migrate_logs_fix.py

```diff
@@ -0,0 +1,50 @@
+import sys
+import logging
+
+from website.app import init_app
+from website.models import NodeLog
+from scripts import utils as script_utils
+from modularodm import Q
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(records, dry=False):
+    for log in records:
+        logger.info(
+            'Migrating log - {}, '.format(log._id)
+        )
+        count = 0
+        if not dry:
+            log.should_hide = False
+            for node in log.logged:
+                if node != log.node:
+                    node.logs.remove(log)
+                    count += 1
+                    node.save()
+                    log.was_connected_to.append(node)
+
+            log.save()
+
+            logger.info(
+                'Removed {} nodes from log - {}'.format(
+                    count, log._id)
+            )
+
+    logger.info('{}Migrated {} logs'.format('[dry]'if dry else '', len(records)))
+
+
+def get_targets():
+    return NodeLog.find(Q('should_hide', 'eq', True))
```

Comment:
While this is fine, it assumes that our previous script was 100% correct in finding the logs attached to other nodes. Is there any reason not to run this against all NodeLog objects? It would make sure we don't miss anything.